### DISCUSSION TITLE
Can we use Object properties for any IRI, regardless of its RDA Entity Status?

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#458] [Q&A] [answered]
@GordonDunsire @szapoun 

Or, do we need to use canonical properties? When we're using value vocabulary IRIs from RDA Registry or minting concepts.

**Comment by GordonDunsire** [comment]
@CECSpecialistI: Currently there are no object properties for entity attributes in RDA Registry. However, as I mentioned in another thread, the RSC Technical Working Group is considering adding object properties for attributes that can have values from a value vocabulary, for example carrier type. This applies to attribute elements that can have a structured description (controlled term) or identifier taken from a value vocabulary. The vocabulary does not have to be an RDA vocabulary.

I therefore recommend that we use placeholder object properties in triples that have an IRI value. We know what the object property IRIs will be because they are generated from the canonical properties with a regular pattern. The actual object properties are likely to be added to the Registry before the transform is published.

This applies to RDA value vocabularies and minted concept IRIs; I have recommended to the Technical Working Group that the range of all such object properties is skos:Concept. This is a direct result of the mutual influence between the project and RDA: RDA values are represented as skos:Concepts, so the project takes a compatible approach with minting its own IRIs; conversely, RDA Registry will assume that RDA and external vocabularies will be modelled with skos:Concept.

**Reply by CECSpecialistI** [reply]
Is this decision a sure thing, or is it still under consideration? It is fairly simple to replace canonical properties with object IRIs when the set of revised IRIs comes out. Changing them early, finding out that the RSC changed course, and then picking out which ones need to be changed back to canonical, would be a more labor-intensive task. It might be better to wait until the RSC makes a firm decision if this is in an early consideration phase?

**Reply by szapoun** [reply]
@GordonDunsire This is really interesting. Will it be implemented for properties that are not related to RDA value vocabularies, e.g., subject?

**Reply by GordonDunsire** [reply]
@CECSpecialistI: I have asked the RSC Technical Working Group for information on a likely timescale for updating the RDA Registry.

@szapoun: The current thinking is that there should be an RDA object property for all attribute properties that accommodate a structured description, even if RDA does not provide a value vocabulary. A structured description of an attribute value implicitly requires a controlled terminology; that is, a value vocabulary.

**Comment by CECSpecialistI** [comment]
@GordonDunsire I seem to remember we got an answer to this question at some point, but I can't find it now. Am I misremembering, or have these object properties been added to the RDA registry?

**Reply by GordonDunsire** [reply]
@CECSpecialistI: This was confirmed in an Update during the January 15th, 2025 call.

**Reply by CECSpecialistI** [reply]
Thank you @GordonDunsire ! I knew it was somewhere.

**Reply by CECSpecialistI** [reply, marked as answer]
Answer to this discussion: https://github.com/uwlib-cams/MARC2RDA/wiki/2025-Meeting-Minutes#updates-10-7
